19.12/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM redis:5.0.7-alpine
+
+RUN apk add --no-cache bash
```

Comment:
See #3.

Reply:
Do you mean delete this line?

Reply:
I have removed this line but after building and run locally for Slergoth, its not working